src/biogeophys/SoilHydrologyMod.F90

```diff
@@ -2490,6 +2495,176 @@ subroutine RenewCondensation(bounds, num_hydrologyc, filter_hydrologyc, &
      end associate
 
    end subroutine RenewCondensation
+!#8
+   !-----------------------------------------------------------------------
+   subroutine CalcAvailableUnconfinedAquifer(bounds, num_hydrologyc, filter_hydrologyc, &
+        soilhydrology_inst, soilstate_inst, waterdiagnosticbulk_inst) 
+     !
+     ! !DESCRIPTION:
+     ! Calculate water in unconfined aquifer (i.e. soil column) that
+     ! is available for groundwater recharge.
+     !
```

Comment:
Thanks a lot @swensosc - this looks good.

Can you please add a comment saying which options this is / is not compatible with? I know there are checks of this elsewhere in the code, but I think it would help to have this noted in the top-level documentation of this function, too.

Specifically, I'm thinking (but would like your confirmation):

- The removal of irrigation from the confined aquifer assumes use_aquifer_layer is false.

- The removal of irrigation from the unconfined aquifer follows the same pattern used in LateralFlowPowerLaw. Does this implicitly assume that we're using a particular formulation for the aquifer layer and/or other parameterizations? If so, it would be good to note that here.

(Maintaining some semblance of sanity with a multi-parameterization model is hard!)